cabotage/scripts/create_admin.py

```diff
@@ -0,0 +1,27 @@
+from ..server import create_app, db
```

Comment:
can these imports be non-relative?

```suggestion
from cabotage.server import create_app, db
```